UiPath.FreeRdpClient/UiPath.FreeRdpWrapper/FreeRdpWrapper.cpp

```diff
@@ -255,10 +281,10 @@ namespace FreeRdpClient
 		auto connectResult = freerdp_connect(instance);
 		if (connectResult)
 		{
-			auto eventName = L"Global\\" + (_bstr_t)(rdpOptions->ClientName);
-			auto lpData = transport_start(context, eventName);
+			auto lpData = transport_start(context, rdpOptions);
 			if (lpData)
 			{
+				auto eventName = lpData->getEventName();
```

Comment:
Does the `instance_data` instance ever get destroyed in this scenario?

Reply:
when the started thread exits